input/ch.fhir.ig.ch-epr-mhealth.xml

```diff
@@ -45,14 +45,14 @@
     <version value="2.1.0"/>
   </dependsOn>
   <dependsOn id="ihepixmfhir">
-    <uri value="http://profiles.ihe.net/ITI/PIXm/ImplementationGuide/IHE.ITI.PIXm"/>
-    <packageId value="IHE.ITI.PIXm"/>
+    <uri value="http://profiles.ihe.net/ITI/PIXm/ImplementationGuide/ihe.iti.pixm"/>
+    <packageId value="ihe.iti.pixm"/>
     <version value="3.0.0"/>
   </dependsOn>
   <dependsOn id="ihemhdfhir">
-    <uri value="http://profiles.ihe.net/ITI/MHD/ImplementationGuide/ihe.mhd.fhir"/>
-    <packageId value="ihe.mhd.fhir"/>
-    <version value="4.0.2"/>
+    <uri value="https://profiles.ihe.net/ITI/MHD/ImplementationGuide/ihe.iti.mhd"/>
+    <packageId value="ihe.iti.mhd"/>
+    <version value="current"/>
```

Comment:
current ist not good, need to use a published version, should be by end of this week published.

Reply:
done